docs/product-business-resource-planner-guide/src/main/asciidoc/optimizer-download-proc.adoc

```diff
@@ -1,23 +1,28 @@
 [id='optimizer-download-proc']
 = Download {PLANNER}
 
-{PLANNER} is production ready.
-The API is stable but backward incompatible changes can occur.
+{PLANNER} is production ready, and can be used to optimize business solutions in {CENTRAL}, in your preferred IDE, or on OpenShift.
```

Comment:
Are we sure about this statement? "Production ready"? Does not make much sense.

Reply:
from the upstream version, I'll remove it here